cmd/tools/releaser/internal/fs/fs.go

```diff
@@ -0,0 +1,63 @@
+package fs
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"go.uber.org/zap"
+)
+
+// Client implements Interface
+type Client struct {
+	logger *zap.Logger
+}
+
+func NewFileSystemClient(logger *zap.Logger) *Client {
+	return &Client{logger: logger}
+}
+
+func (f *Client) FindGoModFiles(root string) ([]string, error) {
+	f.logger.Debug("Finding go.mod files", zap.String("root", root))
+	var goModFiles []string
+
+	err := filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
```

Comment:
```suggestion
		if err != nil {
			return err
		}
		if info.Name() == ".git" {
			return fs.SkipDir
		}
```
probably, so it doesn't find any "happens to be named go.mod" stuff in the git folder.
idls might be useful here too.

Reply:
Simplified to just parsing go.work.